src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -14,6 +14,15 @@
 
 @import '../variables';
 
+* {
+  font-size: 14px;
```

Comment:
Still not fixed. 

Just apply .md-body-1 class to the deploy form. See more at: https://material.angularjs.org/latest/CSS/typography